lib/hooks/constants.js

```diff
@@ -2,8 +2,8 @@ export const CAPABILITIES_LIMIT = 5000;
 export const USERS_BY_ROLE_ID_QUERY_KEY = 'user-role-data';
 
 export const PC_SHARING_DETAILS_KEYS = {
-  createPoliciesPCId: 'createPoliciesPCId',
-  updatePoliciesPCId: 'updateSettingsPCId',
-  createRolesPCId: 'createRolesPCId',
-  updateRolesPCId: 'updateRolesPCId',
+  createPoliciesPCId: 'createPCId',
+  updatePoliciesPCId: 'updatePCId',
+  createRolesPCIds: 'createPCIds',
+  updateRolesPCIds: 'updatePCIds',
```

Comment:
will it also work with sharing settings? because currently we have different contracts from mod-consortia and mod-consortia-kyacloak regarding sharing settings

Reply:
this repo is not connected to sharing settings - it's only about roles and policies

Reply:
but contract changed also for sharing setting, it means  we need change in that repo that is connected with sharing setting, but we can not easily change because also need to change contract in old mod-consortia